types/html-replace-webpack-plugin/tslint.json

```
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-class": false
+    }
+}
```

```suggestion
{
    "extends": "dtslint/dt.json"
}
```
can this be replaced with inline exlclusion? (there is only one place), thx!
https://github.com/DefinitelyTyped/DefinitelyTyped#linter-tslintjson